scripts/lighthouseRun.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e

```

Subtle bug - bash scripts won't exit on error by default which means that if the first lighthouse test fails but the second succeeds, the script as a whole would succeed. `set -e` will make the script exit as soon as it finds an error and make the script fail

Reply:
Nice, I didn't know that!
Cheers again